angrmanagement/ui/main_window.py

```diff
@@ -130,7 +130,7 @@ def progress(self, v):
     #
 
     def _open_mainfile_dialog(self):
-        file_path, _ = QFileDialog.getOpenFileName(self, "Open a binary", ".",
+        file_path, _ = QFileDialog.getOpenFileName(self, "Open a binary", "",
```

Comment:
I can no longer get the dialog to show. Undoing this change doesn't fix it either but something broke for me. I don't have Qt debugging set up but I get no errors from Python... it just silently fails and returns an empty string to `file_path`.

Also, maybe something like `QDir.home().absolutePath()` as a default location?

Reply:
Derp. I always use my CHESS plugin so I haven't actually tried loading a file normally in forever. This is apparently due to my use of PyCharm via snap with a newer PySide2 package [(bug report)](https://bugreports.qt.io/browse/PYSIDE-912).